python/interpret_community/common/constants.py

```diff
@@ -143,6 +143,12 @@ class Spacy(object):
     TAGGER = 'tagger'
 
 
+class Scipy(object):
+    """Provide scipy related constants."""
+
+    CSR_FORMAT = 'csr'
+
```

Comment:
Defining a class for a single constant seems to be too much of an overhead. Specially, since this will be a supported constant so as to not break backward compat. Also, eye() take 'csr' string and they should support that constant. Can we just use that as is?

Reply:
For the record I like the idea of separating all of these constants with classes. If there are additional Scipy constants they can be added here and if those constants also apply to numpy or sklearn interfaces additional constants can be added to those libraries' constant classes as well.